framework/src/fs.js

```diff
@@ -13,67 +13,67 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
+/* eslint-disable consistent-return */
```

Comment:
Please never disable the lint directives globally. Please disable it on a case-by-case basis.